rp2040-hal/src/clocks/mod.rs

```diff
@@ -101,6 +150,30 @@ pub trait ClockSYSClockAuxSource {
     /// Set System Clock as source.
     fn set_clksys_auxsrc(&mut self);
 }
+
+/// For clocks that can have the USB Clock as an auxilliary source.
+pub trait ClockUSBClockAuxSource {
+    /// Set USB Clock as source.
+    fn set_clkusb_auxsrc(&mut self);
+}
+
+/// For clocks that can have the USB Clock as an auxilliary source.
```

Comment:
Copy paste error on the comment for the ADC, RTC, Ref clock blocks

Reply:
no problem, i will clean them up in a moment

Reply:
Fixed